### ISSUE TITLE
[0.9.2.4] Crash on joining White Tiger (for the first time)

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoIssues#21060]
Today I decided to join White Tiger server again, first time in this iteration. There is something up with the server, but on top of that, I ran into a crash while trying to join the server:

[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6186700/Player.log)
[Crash_2021-03-23_032555072.zip](https://github.com/StrangeLoopGames/EcoIssues/files/6186706/Crash_2021-03-23_032555072.zip)

**Comment by IcedForge** [comment]
From player log for search reference
```
NullReferenceException: Object reference not set to an instance of an object.
  at WorldLayerView.GetPercentRaw (Eco.Shared.Math.Vector2i pos) [0x00000] in <00000000000000000000000000000000>:0 
  at BiomeTracker.GetBiomeIndexRaw (Eco.Shared.Math.Vector2i rawPos) [0x00000] in <00000000000000000000000000000000>:0 
  at DecorationParticleSpawner.Update () [0x00000] in <00000000000000000000000000000000>:0 
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```
The dump callstack contains a memory inaccessability, could either of you take a look at it more in depth to see whats went wrong here?

![Screenshot_18.jpg](https://images.zenhubusercontent.com/5f5a7fa59878ed957a861886/5025c45d-241b-4a38-be3f-72f3bbc87e13)

**Comment by mirasrael** [comment]
@ThePiachu do you have this issue every time? Do you have steps to reproduce it? It seems like an crash somewhere inside of Unity library and not much we can get from this crash log. I doesn't make sense to report it to Unity as well, because it is running not latest Unity version. If you can reproduce it then we can make build on latest Unity version and if it still crashes then report it. Otherwise nothing we can do here.

@IcedForge WorldLayer error doesn't seems related to this, but you can create separate issue for @gzatravkin team, because they are working on minimap and world layers as far as I remember.

**Comment by ThePiachu** [comment]
I got it once only, and it was around the time WT server had some issues. I've been able to log into White Tiger since then no problem.

**Comment by WeaselDog** [comment]
Thanks for the reply back I will close as not reproduced but if you get this crash again let us know.

**Comment by ThePiachu** [comment]
Got another crash loading into White Tiger:

[Crash_2021-04-08_071835621.zip](https://github.com/StrangeLoopGames/EcoIssues/files/6276798/Crash_2021-04-08_071835621.zip)

[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6276800/Player.log)

Not sure if it's the same one.

**Comment by mirasrael** [comment]
Blocked by https://github.com/StrangeLoopGames/EcoIssues/issues/18653
